libTest/activityIndicator.swift

```diff
@@ -1,22 +1,14 @@
-//
```

Comment:
sorry for removing the copyright doc. add the docs if you want to have them in the code :)